modules/scripting/src/main/java/org/mule/module/scripting/component/Scriptable.java

```diff
@@ -260,34 +260,35 @@ public void populateBindings(Bindings bindings, MuleEvent event)
         bindings.put(BINDING_FLOW_CONSTRUCT, event.getFlowConstruct());
     }
 
-    protected void populateMessageBindings(Bindings bindings, MuleMessage message)
+    protected void populateMessageBindings(Bindings bindings, MuleEvent event)
     {
+        MuleMessage message = event.getMessage();
         if (message == null)
         {
             message = new DefaultMuleMessage(NullPayload.getInstance(), muleContext);
         }
 
-        populateVariablesInOrder(bindings, message);
+        populateVariablesInOrder(bindings, event);
 
-        bindings.put(BINDING_MESSAGE, message);
+        bindings.put(BINDING_MESSAGE, event.getMessage());
```

Comment:
I'm not sure about this: in the past users could set properties using the message. Now that is not possible as they live on the event. Does not this mean that now we need a way to access the event from the bindings?